project.inlang/project_id

```diff
@@ -0,0 +1 @@
+f2fb93bb94a8ecf19f03f37f743e05cf2458b21c987774763b6725a0703a502c
```

Comment:
where did you get the project id? do we need this?

Reply:
The project id is generated by the inlang sdk. It's needed to uniquely identify projects for monorepo setups.